content/intro-to-writing-data.md

```diff
@@ -204,18 +216,22 @@ const PING_PROGRAM_DATA_ADDRESS =  new web3.PublicKey('Ah9K7dQ8EHaZqcAsgBW8w37yN
 Now, in the `sendPingTransaction()` function, let’s create a new transaction, then initialize a `PublicKey` for the program account, and another for the data account.
 
 ```tsx
-const transaction = new web3.Transaction()
-const programId = new web3.PublicKey(PING_PROGRAM_ADDRESS)
-const pingProgramDataId = new web3.PublicKey(PING_PROGRAM_DATA_ADDRESS)
+async function sendPingTransaction(
+  connection: web3.Connection,
+  payer: web3.Keypair
+) {
+    const transaction = new web3.Transaction()
+    const programId = new web3.PublicKey(PING_PROGRAM_ADDRESS)
+    const programData = new web3.PublicKey(PING_PROGRAM_DATA_ADDRESS)
+}
 ```
 
 Next, let’s create the instruction. Remember, the instruction needs to include the public key for the Ping program and it also needs to include an array with all the accounts that will be read from or written to. In this example program, only the data account referenced above is needed.
 
 ```typescript
 const transaction = new web3.Transaction()
-
 const programId = new web3.PublicKey(PING_PROGRAM_ADDRESS)
-const pingProgramDataId = new web3.PublicKey(PING_PROGRAM_DATA_ADDRESS)
+const programData = new web3.PublicKey(PING_PROGRAM_DATA_ADDRESS)
```

Comment:
The rename makes this less relevant. This more the 'ID' or 'address' for the program data rather than the actual data, which `programData` would imply.